torch/fx/traceback.py

```diff
@@ -0,0 +1,40 @@
+from contextlib import contextmanager
+from typing import Optional
+from ._compatibility import compatibility
+
+__all__ = ['override_stack_trace', 'current_stack_trace', 'is_stack_trace_overridden']
+
+active_interpreter = None
+is_overriden = False
```

Comment:
overridden

Reply:
It seems like is_overridden is always True when active interpreter is not None, so you don't need both variables?